### PR TITLE
LookoutCASB: Initial version

### PR DESCRIPTION
**Pull request by Dhiraj3**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [ x] Documentation

### CODE REVIEW

**Comment by marcellarichmond** [comment]
@mchasepan CipherCloud / Lookup pull request!

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
+[
+	{
+		"id": "CipherCloud Anomaly",
+		"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:4]:
```suggestion
		"version": -1,
```

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+		"default": false,
+		"autorun": true,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0
+	},
+	{
+		"id": "CipherCloud Violation",
+		"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:32]:
```suggestion
		"version": -1,
```

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+		"default": false,
+		"autorun": true,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 1
+	},
+	{
+		"id": "Dhiraj",
+		"version": 2,
```
Comment (adi88d) [inline Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:60]:
```suggestion
		"version": -1,
```

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+		"default": false,
+		"autorun": true,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0
+	},
+	{
+		"id": "Lookout CASB Anomaly",
+		"version": 2,
```
Comment (adi88d) [inline Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:88]:
```suggestion
		"version": -1,
```

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+		"default": false,
+		"autorun": true,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0
+	},
+	{
+		"id": "Lookout CASB Violation",
+		"version": 2,
```
Comment (adi88d) [inline Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:116]:
```suggestion
		"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 6]
```diff
@@ -0,0 +1,887 @@
+{
+	"incidentFields": [
+		{
+			"id": "incident_actiontype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:5]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 7]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_activitytype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:49]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 8]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_anomalyname",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:93]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 9]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_anomalytype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:137]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 10]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_appname",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:181]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 11]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_cloudname",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:225]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 12]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_cloudtype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:269]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 13]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_contentname",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:313]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 14]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_currentcity",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:357]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 15]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_currenteventid",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:401]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 16]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_currenttimestamp",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:445]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 17]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_eventtype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:489]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 18]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": "8af8cb1700ce158bd3918d3d0c6eb0f6"
+		},
+		{
+			"id": "incident_policyname",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:536]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 19]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_previouscity",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:580]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 20]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_previouseventid",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:624]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 21]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_previoustimestamp",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:668]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 22]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_scantype",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:712]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 23]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_timestamp",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:756]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 24]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_useremail",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:800]:
```suggestion
			"version": -1,
```

File: Packs/LookoutCASB/IndicatorFields/incidentfields.json [hunk 25]
```diff
@@ -0,0 +1,887 @@
...
+			"columns": null,
+			"defaultRows": null,
+			"sla": 0,
+			"threshold": 72,
+			"breachScript": "",
+			"validatedError": ""
+		},
+		{
+			"id": "incident_violation",
+			"version": 1,
```
Comment (adi88d) [inline Packs/LookoutCASB/IndicatorFields/incidentfields.json:844]:
```suggestion
			"version": -1,
```

**Review by adi88d — CHANGES_REQUESTED** [review]
@Dhiraj3 Thank you for your contribution! it looks great, few notes:
- Please split the files `incidentfields.json` and `customIncidentTypes.json`, each field and type should be in separate file.
- Update the description for end_time and start_time args with date format for example(i.e. ISO8601 as 2020-01-01T00:11:22Z).
- Add a readme file for the integration and for `Lookout-CASB-User-Risk-PlayBook` using [generate-docs](https://github.com/demisto/demisto-sdk#generate-docs) command.
- update `CASBIntegration_image.png` file - Logo should match the requirements (120x50 px PNG, transparent background, <10kB in Size).
- Add [unitests](https://xsoar.pan.dev/docs/integrations/unit-testing#write-your-unit-tests) to the integration.
- Your code must pass linters see [this](https://xsoar.pan.dev/docs/integrations/linting) article for more info and [lint](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/lint/README.md) command.

see my other comment below

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 26]
```diff
@@ -0,0 +1,183 @@
...
+  id: Lookout CASB
+  version: -1
+name: Lookout CASB
+display: Lookout CASB
+category: Data Enrichment & Threat Intelligence
+description: Lookout CASB Integration
+configuration:
+- display: Server URL (e.g. https://management-server.ciphercloud.com)
+  name: url
+  defaultvalue: https://krypton-ms.next.ciphercloud.in
```
[the remark this answers is not in the document]
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:11]: fixed

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 27]
```diff
@@ -0,0 +1,183 @@
...
+  type: 13
+  required: false
+- display: Maximum number of incidents per fetch
+  name: max_fetch
+  defaultvalue: "1000"
+  type: 0
+  required: false
+- display: Client Id
+  name: client_id
+  defaultvalue: cortexdev
```
[the remark this answers is not in the document]
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:31]: fixed

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 28]
```diff
@@ -0,0 +1,183 @@
...
+  type: 0
+  required: true
+- display: Client Secret
+  name: client_secret
+  defaultvalue: XFrGiFHrQ8cG6a2ZMuYiKWBhf+4tLO7wg2aAdnyhDiA=
```
[the remark this answers is not in the document]
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:36]: fixed

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json (hunk 1 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:4]: fixed

File: Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json (hunk 2 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/IncidentTypes/customIncidentTypes.json:32]: fixed this and all version comments.

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py [hunk 29]
```diff
@@ -0,0 +1,718 @@
...
+    }
+
+    demisto.setIntegrationContext(integration_context)
+    return auth_token
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client, event_type: str) -> str:
```
[the remark this answers is not in the document]
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py:261]: Yes this was used when testing.

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 30]
```diff
@@ -0,0 +1,183 @@
...
+      description: 'Application info'
+      type: string
+    - contextPath: CipherCloud.Info.Device
+      description: 'Device info'
+      type: string
+    - contextPath: CipherCloud.Info.Content
+      description: 'Content info'
+      type: string
+    description: Get information about user or content.
+  - name: lookout-profile-user
```
[the remark this answers is not in the document]
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml]: fixed.

**Comment by adi88d** [comment]
@Dhiraj3
- Update the description for end_time and start_time params with date format for example(i.e. ISO8601 as 2020-01-01T00:11:22Z).
- Add a readme file for the integration using [generate-docs](https://github.com/demisto/demisto-sdk#generate-docs) command.
- Add [unitests](https://xsoar.pan.dev/docs/integrations/unit-testing#write-your-unit-tests) to the integration.
- Renamae the incident fields name to starts with `incidentfield-` (for example: `incidentfields-LookoutCASB-actiontype.json` to `incidentfield-LookoutCASB-actiontype.json`)
- Please add readable_output to your commands (see this [page](https://xsoar.pan.dev/docs/reference/api/common-server-python#commandresults))

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py [hunk 31]
```diff
@@ -0,0 +1,746 @@
...
+
+        elif demisto.command() == 'lookout-get-violations':
+            violations = fetch_events(
+                client=client,
+                event_type=EVENT_TYPE_VIOLATION,
+                start_time=start_time,
+                end_time=end_time,
+                max_results=max_results
+            )
+
```
Comment (adi88d) [inline Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py:636]: readable_output =  tableToMarkdown('Violations', violations, removeNull=True, headerTransform=string_to_table_header)

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py [hunk 32]
```diff
@@ -0,0 +1,746 @@
...
+            cmd_result = CommandResults(
+                outputs_prefix='CipherCloud.Violation',
+                outputs_key_field=['eventId'],
+                outputs=violations
```
Comment (adi88d) [inline Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.py:640]:
```suggestion
                outputs=violations,
                readable_output=readable_output
```

**Comment by Dhiraj3** [comment]
Addressed few review comments. Working on remain comments.

File: Packs/LookoutCASB/README.md [hunk 33]
```diff
@@ -0,0 +1,14 @@
+### Lookout CASB Integration
```
Comment (mchasepan) [inline Packs/LookoutCASB/README.md:1]: This content is a copy of the integration readme. Please delete this content and add a PACK readme (required) that talks about the value of the pack, what it does, what Lookout does, links to your site, etc. Let us know if you need help or example.

**Comment by adi88d** [comment]
@Dhiraj3 Did you finished working on my comments?

**Comment by Dhiraj3** [comment]
Fixed most of the comments. But still need to address few comments, and validate.

File: Packs/LookoutCASB/README.md (hunk 33 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/README.md:1]: Sure will change it.

**Comment by adi88d** [comment]
@Dhiraj3 Did you finished working on my comments?

**Comment by Dhiraj3** [comment]
Cloud you please re-review - I have addressed all the comments except for additional tests and rename file.

File: Packs/LookoutCASB/Playbooks/Lookout-CASB-User-Risk-PlayBook_README.md [hunk 34]
```diff
@@ -0,0 +1,29 @@
+Playbook for user risk update based on violations.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+This playbook does not use any integrations.
```
Comment (adi88d) [inline Packs/LookoutCASB/Playbooks/Lookout-CASB-User-Risk-PlayBook_README.md:10]:
```suggestion
* CASBIntegration
```

File: Packs/LookoutCASB/Playbooks/Lookout-CASB-User-Risk-PlayBook_README.md [hunk 35]
```diff
@@ -0,0 +1,29 @@
...
+---
+There are no inputs for this playbook.
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Lookout-CASB-User-Risk-PlayBook](Insert the link to your image here)
```
Comment (adi88d) [inline Packs/LookoutCASB/Playbooks/Lookout-CASB-User-Risk-PlayBook_README.md:29]:
```suggestion
![Lookout-CASB-User-Risk-PlayBook](../../doc_files/Lookout-CASB-User-Risk.png)
```

File: Packs/LookoutCASB/README.md (hunk 33 above)
Comment (adi88d) [reply Packs/LookoutCASB/README.md:1]: please update this file

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 36]
```diff
@@ -0,0 +1,187 @@
...
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Start Time
+  name: start_time
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: End Time
```
Comment (adi88d) [inline Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:59]:
```suggestion
- display: End Time(in ISO8601 format)
```

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 37]
```diff
@@ -0,0 +1,187 @@
...
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Start Time
```
Comment (adi88d) [inline Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:54]:
```suggestion
- display: Start Time(in ISO8601 format)
```

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml [hunk 38]
```diff
@@ -0,0 +1,187 @@
...
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Maximum number of incidents per fetch
+  name: max_fetch
```
Comment (adi88d) [inline Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:25]:
```suggestion
  name: max_fetch
```

this variable only configured in the yml file but you dont use it at all, please limit the fetched incidents with this value

**Review by adi88d — COMMENTED** [review]
@Dhiraj3 few more notes:

- Add unitests to the integration.
- Add a readme file for the integration using [generate-docs](https://github.com/demisto/demisto-sdk#generate-docs) command.
- Add the command `lookout-add-user`
- Add limit argument to the commands:
   - lookout-get-anomalies
   - lookout-get-events
   - lookout-get-information
   - lookout-get-violations
   
see my other comment below

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml (hunk 38 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:25]: max_fetch is optional. There is a max limit at the server side. This field is provided for client as optional if client would like to limit the records.

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml (hunk 36 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:59]: updated

File: Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml (hunk 37 above)
Comment (Dhiraj3) [reply Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml:54]: updated

**Comment by Dhiraj3** [comment]
Our use case does not need command `lookout-add-user`

**Comment by adi88d** [comment]
@Dhiraj3 please fix my last comments:

- Add unitests to the integration.
- Add a readme file for the integration using [generate-docs](https://github.com/demisto/demisto-sdk#generate-docs) command.
- Add limit argument to the commands:
   - lookout-get-anomalies
   - lookout-get-events
   - lookout-get-information
   - lookout-get-violations
- Please use the value from `max_fetch` parameter inside the python file to limit the fetched incidents.

**Comment by ShahafBenYakir** [comment]
Closing as we didn't hear back from the partner.

**Pull request closed without merging**